.github/workflows/container.yml

```diff
@@ -0,0 +1,77 @@
+name: Container Image Builds
+
+on:
+  push:
+    branches: [ oldstable ]
+    tags: ["v*"]
+  workflow_dispatch:
+
+jobs:
+  images:
+    name: Build and upload container images
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Gather container image tags
+        uses: greenbone/actions/container-image-tags@v1
+        id: container
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push Container image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.container.outputs.image-tags }}
+          file: .docker/oldstable.Dockerfile
+
+  trigger-openvas:
+    needs: images
+    name: "Trigger docker "
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger openvas container image build
+        run: |
+          curl -X POST https://api.github.com/repos/greenbone/openvas-scanner/dispatches \
+          -H 'Accept: application/vnd.github.everest-preview+json' \
+          -u greenbonebot:${{ secrets.GREENBONE_BOT_TOKEN }} \
+          --data '{"event_type": "update-images"}'
```

Comment:
```suggestion
  trigger-related-projects:
    needs: images
    name: "Trigger update docker images in related projects"
    runs-on: ubuntu-latest
    steps:
      - name: Trigger oldstable openvas-scanner container image build
        run: |
          curl -X POST https://api.github.com/repos/greenbone/openvas-scanner/dispatches \
          -H 'Accept: application/vnd.github.everest-preview+json' \
          -u greenbonebot:${{ secrets.GREENBONE_BOT_TOKEN }} \
          --data '{"event_type": "update-oldstable-images"}'
      - name: Trigger oldstable gvmd container image build
        run: |
          curl -X POST https://api.github.com/repos/greenbone/gvmd/dispatches \
          -H 'Accept: application/vnd.github.everest-preview+json' \
          -u greenbonebot:${{ secrets.GREENBONE_BOT_TOKEN }} \
          --data '{"event_type": "update-oldstable-images"}'
      - name: Trigger oldstable openvas container image build
        run: |
          curl -X POST https://api.github.com/repos/greenbone/gsad/dispatches \
          -H 'Accept: application/vnd.github.everest-preview+json' \
          -u greenbonebot:${{ secrets.GREENBONE_BOT_TOKEN }} \
          --data '{"event_type": "update-oldstable-images"}'
```

Something like this?
We need to differ the event_type for the different versions. Also we might want to rebuild gsad and gvmd, too.
(need to add this to stable, main)

Reply:
looks good to me.